refinery/config/urls.py

```diff
@@ -31,7 +31,7 @@
                                   ProtocolResource, PublicationResource,
                                   StudyResource)
 from data_set_manager.urls import data_set_manager_router
-from user_files_manager.urls import user_files_router
+from user_files_manager.urls import user_files_router, user_files_url
```

Comment:
Shouldn't this import be below: `from file_store.urls import file_store_router` ? (not sure how isort is doing this)